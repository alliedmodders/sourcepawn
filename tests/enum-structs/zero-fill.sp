native void printnums(...);

enum struct Numbers
{
	int first;
	int second;
	int third;
}

static Numbers list[] = {
	{ 1, 2 },
	{ 4, 5, 6 },
	{ 7 },
}

public void main()
{
	for (int i = 0; i < sizeof(list); i++) {
		printnums(i, list[i].first, list[i].second, list[i].third);
	}
}
