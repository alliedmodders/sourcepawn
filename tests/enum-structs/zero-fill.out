0, 1, 2, 0
1, 4, 5, 6
2, 7, 0, 0
